pages/shared/data/tools.yaml

```diff
@@ -316,3 +316,13 @@ providers:
       fit: contain
     formats:
       - email
+  - name: Mailgun
+    url: https://mailgun.com/
+    type: Email Service
+    image:
+      src: /static/img/tools/mailgun.png
+      width: 1223
+      height: 450
```

Comment:
width and height seems to be just duplicated from the entry above.
Please make sure to have the correct image size as soon as the official logo is also uploaded.